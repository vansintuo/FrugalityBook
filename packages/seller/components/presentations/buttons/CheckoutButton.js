import { Button } from "@mui/material";
import React, { Children } from "react";
import ShoppingCartCheckoutIcon from "@mui/icons-material/ShoppingCartCheckout";

const CheckoutButton = ({ onClick, children, style }) => {
  return (
    <Button
      sx={{ backgroundColor: "#00BDD7", color: "white",'&:hover':{color:'#00bdd7'} }}
      variant="outline"
      onClick={onClick}
      startIcon={<ShoppingCartCheckoutIcon />}
      style={{...style}}
    >
      {children}
    </Button>
  );
};

export default CheckoutButton;
