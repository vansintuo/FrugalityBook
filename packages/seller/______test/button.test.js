import CheckoutButton from "../components/presentations/buttons/CheckoutButton";
import Enzyme, { shallow } from "enzyme";

it("button match with snapshot", () => {
  const checkoutButton = shallow(<CheckoutButton />);
  expect(checkoutButton).toMatchSnapshot();
});
