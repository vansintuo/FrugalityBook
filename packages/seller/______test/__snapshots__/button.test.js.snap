// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button match with snapshot 1`] = `ShallowWrapper {}`;
